platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/iss/internal/ConsensusHashFinder.java

```diff
@@ -279,6 +276,9 @@ public void writePartitionData(final StringBuilder sb) {
             sb.append("  partition hash: ")
                     .append(partition.getHash().toMnemonic())
                     .append("\n");
+            sb.append("  partition hash (unabbreviated): ")
```

Comment:
If you take my suggestion above for the labels, be sure to rename this too.

Reply:
Done